### ISSUE TITLE
`inert` property is missing from `domProps`

### ISSUE THREAD

**Issue by jquesada2016** [rescript-lang/rescript-react#124]
I was trying to add `inert` property to an element for accessibility, but was surprised to see the binding was missing.

**Comment by mediremi** [comment]
I've created a PR for adding `inert` to `JsxDOM.domProps` in the main repo: https://github.com/rescript-lang/rescript/pull/7326
